helmfile/charts/grafana-ops/dashboards/prometheus-blackbox-exporter-dashboard.json

```diff
@@ -0,0 +1,1089 @@
+               },
+               {
+                  "selected" : true,
+                  "text" : "10s",
+                  "value" : "10s"
+               },
+               {
+                  "selected" : false,
+                  "text" : "30s",
+                  "value" : "30s"
+               },
+               {
+                  "selected" : false,
+                  "text" : "1m",
+                  "value" : "1m"
+               },
+               {
+                  "selected" : false,
+                  "text" : "10m",
+                  "value" : "10m"
+               },
+               {
+                  "selected" : false,
+                  "text" : "30m",
+                  "value" : "30m"
+               },
+               {
+                  "selected" : false,
+                  "text" : "1h",
+                  "value" : "1h"
+               },
+               {
+                  "selected" : false,
+                  "text" : "6h",
+                  "value" : "6h"
+               },
+               {
+                  "selected" : false,
+                  "text" : "12h",
+                  "value" : "12h"
+               },
+               {
+                  "selected" : false,
+                  "text" : "1d",
+                  "value" : "1d"
+               },
+               {
+                  "selected" : false,
+                  "text" : "7d",
+                  "value" : "7d"
+               },
+               {
+                  "selected" : false,
+                  "text" : "14d",
+                  "value" : "14d"
+               },
+               {
+                  "selected" : false,
+                  "text" : "30d",
+                  "value" : "30d"
+               }
+            ],
+            "query" : "5s,10s,30s,1m,10m,30m,1h,6h,12h,1d,7d,14d,30d",
+            "refresh" : 2,
+            "skipUrlSync" : false,
+            "type" : "interval"
+         },
+         {
+            "current" : {
+               "selected" : false,
+               "text" : "All",
+               "value" : "$__all"
+            },
+            "datasource" : {
+               "uid" : "${DS_PROMETHEUS}"
+            },
+            "definition" : "",
+            "hide" : 0,
+            "includeAll" : true,
+            "multi" : true,
+            "name" : "target",
+            "options" : [],
+            "query" : {
+               "query" : "label_values(probe_success, instance)",
+               "refId" : "prometheus-sc-target-Variable-Query"
+            },
+            "refresh" : 1,
+            "regex" : "",
+            "skipUrlSync" : false,
+            "sort" : 0,
+            "tagValuesQuery" : "",
+            "tagsQuery" : "",
+            "type" : "query",
+            "useTags" : false
+         },
+         {
+            "current" : {
+               "selected" : false,
+               "text" : "prometheus-sc",
+               "value" : "prometheus-sc"
+            },
```

Comment:
Could you ensure that the default datasource is the default?

Reply:
How do you do that?

Reply:
Updated, I think